packages/create-turbo/src/transforms/update-commands-in-readme.ts

```diff
@@ -0,0 +1,44 @@
+import path from "node:path";
+import fs from "node:fs/promises";
+import { TransformError } from "./errors";
+import type { TransformInput, TransformResult} from "./types";
+
+const meta = {
+  name: "update-commands-in-readme",
+};
+
+export async function transform(args: TransformInput): TransformResult {
+  const { prompts, example } = args;
+
+  const isOfficialStarter =
+    !example.repo ||
+    (example.repo.username === "vercel" && example.repo.name === "turbo");
+
+  if (!isOfficialStarter) {
+    return { result: "not-applicable", ...meta };
+  }
```

Comment:
I think we should not restrict this official starters only - reason being we don't restrict the package manager transform to official starters, so if we're changing the package manager I think we should do a best effort at docs as well.

Reply:
removed isOfficialStarter check